src/Playwright/Playwright.csproj

```diff
@@ -40,11 +40,13 @@
     <None Include=".drivers\linux\LICENSE" Pack="true" PackagePath=".playwright\node" />
     <None Include=".drivers\linux\node" Link=".playwright\node\linux-x64\node" CopyToOutputDirectory="PreserveNewest" Pack="true" PackagePath=".playwright\node\linux-x64" />
     <None Include=".drivers\linux-arm64\node" Link=".playwright\node\linux-arm64\node" CopyToOutputDirectory="PreserveNewest" Pack="true" PackagePath=".playwright\node\linux-arm64" />
-    <None Include=".drivers\mac\node" Link=".playwright\node\mac\node" CopyToOutputDirectory="PreserveNewest" Pack="true" PackagePath=".playwright\node\mac" />
+    <None Include=".drivers\mac\node" Link=".playwright\node\darwin-x64\node" CopyToOutputDirectory="PreserveNewest" Pack="true" PackagePath=".playwright\node\darwin-x64" />
```

Comment:
Ah I see. It is confusion to use different folders for `Include` and `PackagePath`.
`.drivers\mac\node` vs `.playwright\node\darwin-x64`
It would be easier to use the same folder e.g. either `mac` or `darwin-x64` for both of them.

Reply:
my idea was to rewrite our custom naming from Node.js drivers to the standardised .NET naming.